packages/patterns-react/src/patterns/SimpleLongForm/SimpleLink.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import { ArrowRight20 } from '@carbon/icons-react';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Simple Link Component
+ *
+ * @param {*} { link: { href, text, target } }
+ * @returns {*} SimpleLink component
+ */
+/**
+ * Simple Link Component
+ *
+ * @param {object} props props object
+ * @param {string} props.href simple link link
+ * @param {string} props.target simple link text
+ * @returns {*} SimpleLink component
+ */
+const SimpleLink = ({ link: { href, text, target } }) => {
+  return (
+    <a
```

Comment:
Is there any particular reason we're not using `LinkWithIcon`, or even carbon's `Link` component?

Reply:
Replaced with global LinkWithIcon by [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/068ea91c5b762045b64a4a026eecc2b600fa1ce4)